framework/source/class/qx/lang/normalize/Array.js

```diff
@@ -384,49 +384,29 @@ qx.Bootstrap.define("qx.lang.normalize.Array", {
     }
   },
 
+  /**
+   * @lint environmentNonLiteralKey()
```

Comment:
Maybe nitpicking here, but shouldn't this lint exception be more "focused"? Like:
```
  /*
   * @lint environmentNonLiteralKey(key)
   */
```
[Note the **key**]